tests/decimal_tests.rs

```diff
@@ -3473,7 +3473,7 @@ fn declarative_ref_dec_sum() {
     assert_eq!(sum, Decimal::from(45))
 }
 
-#[cfg(feature = "postgres")]
+#[cfg(feature = "db-postgres")]
 #[test]
 fn to_from_sql() {
```

Comment:
You're absolutely right about this - thank you. I think there is one more small change to ensure it is picked up by CI:

```suggestion
fn postgres_to_from_sql() {
```